fineract-provider/build.gradle

```diff
@@ -529,9 +533,26 @@ bootWar {
     mainClassName = 'org.apache.fineract.ServerWithMariaDB4jApplication'
 }
 
+//This is for code and swagger UI generation
+dependencies {
+    swaggerCodegen 'io.swagger:swagger-codegen-cli:2.4.2'             // Swagger Codegen V2
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.5'  // or Swagger Codegen V3
+    swaggerCodegen 'org.openapitools:openapi-generator-cli:3.3.4'     // or OpenAPI Generator
+    swaggerUI 'org.webjars:swagger-ui:3.10.0'
+}
+
+swaggerSources {
+    fineract {
+        inputFile = file('src/main/resources/swagger-ui/response.json')
+        code {
+            language = 'java'
+        }
+    }
+}
+
```

Comment:
@kangbreder  Can you please clarify about these changes?
I am not sure how this code will help us?.

Reply:
> @kangbreder Can you please clarify about these changes?
> I am not sure how this code will help us?.

@Sanyam96  have you tried using this to see how it works? Are you sure you really want me to remove all changes made on the build.script. Are you sure about that?

Reply:
> @kangbreder Can you please clarify about these changes?
> I am not sure how this code will help us?.

@Sanyam96 I think i had explained the usage of this in my gist https://gist.github.com/kangbreder/034f47e2e8015cee10b48b7c5f1b8df1, generating client libraries and language-specific SDKs in languages such as java and angular from swagger specification file. This was part of the goals of the project listed in https://mifosforge.jira.com/wiki/spaces/RES/pages/812810251/Google+Summer+of+Code+2019+Ideas. 

I clarified the usage of this this to @vorburger the other day. I had removed the other plugin which was added for generation of spec file during build. Since that did not work, it has been removed it from the PR. 

I suggest you verify that again.
 My ubuntu 18 will not boot.
 I have been trying to solve the problem. Because of that, i have not been able to do the modifications.

Reply:
Thanks for the clarification @kangbreder .

Please do other requested changes other than build.gradle.